study-builder/studies/testboston/i18n/es.conf

```diff
@@ -28,7 +28,7 @@
     "s2_results_p2": "No recibirá los resultados de la prueba de detección de anticuerpos. Presentaremos un resumen de los resultados (sus resultados combinados con los de otros participantes de la investigación) al Departamento de Salud Pública de Massachusetts y otras partes interesadas clave del sector de salud pública. Nuestro objetivo es determinar qué porcentaje de participantes del estudio tienen un anticuerpo positivo y contar con una noción más clara de la prevalencia, o cantidad de personas contagiadas, de COVID-19.",
     "s2_results_p3": "Si lo desea, podemos incluir su correo electrónico en una lista para recibir un boletín de novedades sobre la investigación que estamos haciendo. En el boletín no se anunciarán sus resultados ni los de nadie, sino que se difundirá información que se está recabando sobre COVID-19. Quizás publiquemos los datos que obtengamos en revistas médicas.",
     "s2_sponsor_title": "Patrocinador de la investigación",
-    "s2_sponsor_p1": "Broad Institute de MIT y Harvard.",
+    "s2_sponsor_p1": "Broad Institute of MIT and Harvard.",
```

Comment:
Keeping proper names like `Broad Institute of MIT and Harvard` as-is, just like how we kept `Brigham and Women's Hospital`.